pkg/flow/server.go

```diff
@@ -634,7 +654,71 @@ func (srv *server) publishDemoMessage(subject, msg string) error {
 	if err != nil {
```

Comment:
why are we publishing a demo message? Is that required?

Reply:
no this was just to demo that it's working

Reply:
thanks for pointing this out i removed it.
it's a demo how to use it just as the name suggest, it's not required, was just a placeholder ntil we have something real